src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
+
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public String getColor() {
+        return color;
+    }
```

Comment:
You don't really need getters and setters ffor this task

Reply:
when i did mvn clean package one of the mistakes was to create getters and setters that's why they are here